### ISSUE TITLE
Quick-Command keyboard shortcuts for asetniop

### ISSUE THREAD

**Issue by hunner** [qmk-combos/combos#2]
**tl;dr** An easy way to do keyboard shortcuts with asetniop keyboards like `cmd+s`, `ctrl+v`, `shift+pgup`, or even things like `cmd+ctrl+option+space` in only two or three chords instead of five-plus.

----

### Preamble
My Ginny is in the mail so I've not actually used the QMK asetniop implementation yet, but I had an idea for tackling the dance-routine that is keyboard shortcuts, such as `C-S-z` (`ctrl+super+z`, `cmd+ctrl+z`, or `win+ctrl+z` depending on your fanboi status) and wanted to write it down before I forgot it. I can maybe take a stab at coding it once I get my Ginny to test.

### Problem
Every single keyboard shortcut using modifier keys needs command mode, and every single keyboard shortcut using command mode takes five or more combos.

For example, `C-S-z` takes seven chords: `Command mode (ETNI)`/`Commands (ESIO)`/`ctrl (A)`/`Commands (ESIO)`/`super (T)`/`z (AI)`/`Command mode (ETNI)`.[[1]](#foot1) This is awful for a person with a nearly-100% keyboard-driven interface.

### Proposal
The idea is to put all modifier keys (`shift`, `ctrl`, `alt`, and `super`) on a hold-layer that is enabled by holding both thumbs down and auto-combines with the next keycode. The mapping for ASET keys when both thumbs are held would be `shift`, `ctrl`, `alt`, and `super` respectively (or perhaps swap alt/super to match windows keyboards).

All other keys are generally not used as mods: insert, pgup/pgdn, F-keys, punctuation, letters, numbers, arrows, space/backspace/return, etc. are all "final" keys that may be combined with modifiers and not modifiers themselves, and it is never the case that more than one final key is combined outside of stenography or other chorded-typing engines. (The one exception is `esc`, but it is so rarely used as a mod and not worth including in Quick-Command.)

### Example cases
So how does this stack up for some example shortcuts?
- To get the previously-mentioned `C-S-z` it would be `LRST`/`z`, only two chords instead of seven. This is the keyboard shortcut that re-tiles a floating window in my window manager, used a few times a day.
- `S-s` (cmd+s on mac, commonly used for saving files) instead of `ETNI`/`ESIO`/`T`/`S`/`ETNI` (five chords) it would be `LRT`/`s` (two chords).
- `ctrl+numbers` are how I switch desktop spaces and I use this hundreds of times a day. This would go from six chords to three, eg: `LRS`/`ATNP`/`A` for `C-1`
- `shift+pgdn` is almost a problem as pgdn/up are bound to thumbs in command mode, but this would be `LRA`/`ESIO`/`R`
- How about `shift+space` which is both thumbs and a single combo currently? `LRA`/`R`. This is the only LONGER combo that I'm aware of. Perhaps even pressing both thumbs and no keys could trigger this, but I'd bet that just aborting quick-command would be better.

### Extra credit
The four mods I've mentioned have both left and right versions. rarely needed, but still helpful. We have the `RGT` and `LFT` combo keywords, so if thumbs are pressed left-first, `LRA`, then it would be left-shift. Right-first `RLA` would be right-shift.

<a name="foot1">1</a>: If command mode is sticky then the second `ESIO` combo would come later, but high-number of combos needed still holds.

**Comment by germ** [comment]
Hey hunner, the big issue right now is that I'm backlogged to shit with orders! This has been a bit of a sticking point with Ginny that needs to be ironed out. Right now you can do verbose strokes using command mode (or inline them as their own defined chords in the dictionaries). Less then ideal

I like your proposal, and the basic code to get that working is in there. When your device gets in if you can come up with a layout that's backwards compatible with the original keybinds I'll merge it in (and be grateful for the PR!).

It's good to see someone coming up for solutions for the shortcomings of ASETNIOP! Sorry I can't be more help at the current moment :(
